REFERENCE.md

```diff
@@ -0,0 +1,195 @@
+# Learnosity Node.js-SDK: Reference guide
+
+## Examples
+You can find a complete site with examples of Learnosity integration in our [demo site](http://demos.learnosity.com/).
+
+You can download the entire site or browse the code directly on [GitHub](https://github.com/Learnosity/learnosity-demos/).
+
+## Usage
+
+### Learnosity.init()
+
+The init function is used to create the necessary *security* and *request* details used to integrate with a Learnosity API. Most often this will be a JavaScript object.
+
+The init function takes up to 5 arguments:
+
+ * [string]  service type
+ * [array]   security details (**no secret**)
+ * [string]  secret
+ * [request] request details *(optional)*
```

Comment:
`[string | object]`

Reply:
For discussion.

Reply:
Happy to change something here, if required.